tictactoe.py

```diff
@@ -1,26 +1,63 @@
+empty_squares = [1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
This seems pretty close to being what the [range](https://docs.python.org/3/library/functions.html#func-range) function does, have you considered using that?

Reply:
Oh! I know the range function, but I’ve only seen it used as part of a for loop — so didn’t know how easy it was to use for something like this. 😁🎓